dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.cpp

```diff
@@ -993,41 +988,28 @@ void FindDDLMViaEnumeration(
         const auto currentArchitecture{ AppModel::Identity::GetCurrentArchitecture() };
         if (architecture != currentArchitecture)
         {
-            (void)LOG_HR_MSG(MDD_E_BOOTSTRAP_INITIALIZE_DDLM_SCAN_NO_MATCH,
-                             "Bootstrap.Intitialize: %ls not applicable. Architecture doesn't match current architecture %ls (%ls)",
-                             packageFullName.c_str(), ::AppModel::Identity::GetCurrentArchitectureAsString(), criteria.get());
-            continue;
-        }
-
-        // Do we have a package under consideration?
-        if (!foundAny)
-        {
-            (void)LOG_HR_MSG(MDD_E_BOOTSTRAP_INITIALIZE_DDLM_SCAN_MATCH,
-                             "Bootstrap.Intitialize: %ls is applicable (%ls)",
-                             packageFullName.c_str(), criteria.get());
-            bestFitVersion = version;
-            bestFitPackageFamilyName = packageId.FamilyName();
-            bestFitPackageFullName = packageId.FullName();
-            foundAny = true;
             continue;
         }
 
         // Do we already have a higher version under consideration?
+        // this works for both cases :
+        // 1. find the first package which matches the criteria (because we initiailizue bestfit with 0 version)
+        // 2. find a better match than previously matched best fit
         if (bestFitVersion.Version < version.Version)
         {
-            (void)LOG_HR_MSG(MDD_E_BOOTSTRAP_INITIALIZE_DDLM_SCAN_MATCH,
-                             "Bootstrap.Intitialize: %ls is more applicable (%ls)",
-                             packageFullName.c_str(), criteria.get());
             bestFitVersion = version;
             bestFitPackageFamilyName = packageId.FamilyName();
             bestFitPackageFullName = packageId.FullName();
             continue;
         }
     }
-    THROW_HR_IF_MSG(STATEREPOSITORY_E_DEPENDENCY_NOT_RESOLVED, !foundAny, "Enumeration: %ls", criteria.get());
-    (void)LOG_HR_MSG(MDD_E_BOOTSTRAP_INITIALIZE_DDLM_FOUND,
-                     "Bootstrap.Intitialize: %ls best matches the criteria (%ls) of %d packages scanned",
-                     bestFitPackageFullName.c_str(), criteria.get(), packagesScanned);
+    // if bestFitVersion is 0, the value we initialized it with,  then we didn't find a suitable candidate
+    THROW_HR_IF_MSG(E_FAIL, bestFitVersion.Version == 0, "No suitable version matching criteria is found. Criteria: %ls", criteria.get());
+    {
+        wchar_t printmsg[128];
+        Common::Logging::DebugLog(std::format(L"Bootstrap.Intitialize: {} best matches the criteria ({}) of {} packages scanned",
```

Comment:
is this a generalized roll-up log message to replace all the removed ones above?

Reply:
I decided against it. the log messages were like 
Testing foo package but it didn't match . arch : x86, looking : x64.

those messages add up a lot if you have more than 1 winappsdk package and provide little diagnostic value. There are 5-10 lines of these in VS console output which would confuse developers. Hence, removed them